includes/discovery/sensors/dbm/fabos.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$multiplier = 1;
+$divisor    = 1;
+
+foreach ($pre_cache['fabos_Sfp'] as $index => $entry) {
+    $entPhysicalIndex = substr($index, -2, 2);
```

Comment:
Perhaps this should be `substr($index, strrpos($index, '.'))` in case the number is longer than 2 digits

Reply:
Thanks! Changed